Formula/periphery.rb

```diff
@@ -0,0 +1,19 @@
+class Periphery < Formula
+  desc "Tool to identify unused code in Swift projects"
+  homepage "https://github.com/peripheryapp/periphery"
+  url "https://github.com/peripheryapp/periphery/archive/1.4.0.tar.gz"
+  sha256 "a47920cde9584547d46c8b1b9207ba06577ecb2cb25b21dceaeb5cad8d55a82d"
+
+  bottle do
+  end
+
+  depends_on :xcode => "10.1"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/periphery"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.